IM_calculation/IM/im_calculation.py

```diff
@@ -327,12 +362,16 @@ def write_rows(comps, station, ims, result_dict, big_csv_writer, sub_csv_writer=
     :return: write a single row
     """
     for c in comps:
```

Comment:
How does this work if we just create a pandas dataframe?

Reply:
Converting it ot use a dataframe sounds like a new task. It should be ok though as you can use to_csv to append to existing csvs, as long as we iterate over stations when saving